uiowa/uiowa
# Site customizations for 91st Meridian

Resolves #5601 
Resolves #5595

# To do

- [x] Construct Publication Issue title using volume number, issue number, and season/year (similar to how title is constructed for Person content types
- [x] Fix publication articles showing up on /news page
- [x]  Update article display to match default theming already in place for articles

# How to test
```
ddev blt frontend && ddev blt ds --site 91stmeridian.iwp.uiowa.edu && ddev drush @iwp91stmeridian.local en sitenow_periodical && ddev drush @iwp91stmeridian.local config-split:activate periodical -y && ddev drush @iwp91stmeridian.local cim -y && ddev drush @iwp91stmeridian.local config:import --source ../config/sites/91stmeridian.iwp.uiowa.edu --partial -y && ddev drush @iwp91stmeridian.local cim -y && ddev drush @iwp91stmeridian.local cr && ddev drush @iwp91stmeridian.local uli node/add/publication_issue
```

- `blt frontend`
- `ddev blt ds --site 91stmeridian.iwp.uiowa.edu`
- `ddev drush @iwp91stmeridian.local en sitenow_periodical`
- `ddev drush @iwp91stmeridian.local config-split:activate periodical -y`
- `ddev drush @iwp91stmeridian.local cim`
- Enable site split : `ddev drush @iwp91stmeridian.local config:import --source ../config/sites/91stmeridian.iwp.uiowa.edu --partial && ddev drush @iwp91stmeridian.local cim`
- `ddev drush @iwp91stmeridian.local cr` to get moderation block working. 
- `ddev drush @iwp91stmeridian.local uli node/add/publication_issue`
- Try adding Publication Issues.
- Try adding Articles. Check that filling in the Issue field changes the URL path on save, while leaving the field empty results in our default news/... path.
- Check usual things like user roles and content moderation block working, you could also validate the schema for article and publication issue. 
- Code review also appreciated
